app/serializers/concerns/optional_serialized_attributes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module OptionalSerializedAttributes
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def attribute_if_positive(optional_attribute)
+      attribute optional_attribute, if: -> { object.send(optional_attribute).positive? }
+    end
```

Comment:
Not including an attribute implies this item doesn't have this, but it could additionally imply that all items in the set of items of this type do not have this attribute.

Whereas a null value implies this value for this item is null, all other items in the set of items having this type could or could not have a value for the attribute.

We want to convey the latter, not the former.